jax/_src/numpy/fft.py

```diff
@@ -553,15 +553,158 @@ def _fft_core_2d(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return _fft_core(func_name, fft_type, a, s, axes, norm)
 
 
-@implements(np.fft.fft2)
 def fft2(a: ArrayLike, s: Shape | None = None, axes: Sequence[int] = (-2,-1),
          norm: str | None = None) -> Array:
+  """Compute a two-dimensional discrete Fourier transform along given axes.
+
+  JAX implementation of :func:`numpy.fft.fft2`.
+
+  Args:
+    a: input array. Must have ``a.ndim >= 2``.
+    s: optional length-2 sequence of integers. Specifies the size of the output
+      along each specified axis. ``s`` should have same length as that of ``axes``.
+      If not specified, it will default to the shape of ``a`` along the specified
+      ``axes``.
+    axes: optional length-2 sequence of integers, default=(-2,-1). Specifies the
+      axes along which the transform is computed. Length of ``axes`` must be
```

Comment:
We've already said ``axes`` must be length 2, no need to repeat that

Reply:
Removed the statement about `axes` length.